cms/toolbar/toolbar.py

```diff
@@ -444,6 +444,7 @@ def post_template_populate(self):
         if not self.request.user.is_staff:
             return
         self._call_toolbar('post_template_populate')
+        self._render_object_editable_buttons()
```

Comment:
this should not be done here.
Instead do it in the post_template_populate of the PlaceholderToolbar.
Placeholder toolbar and editable buttons are a cms specific feature that should not be in the generic toolbar class